packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -251,13 +251,17 @@ export const isLandingCompanyEnabled = ({ landing_companies, platform, type }: T
 };
 
 export const getAuthenticationStatusInfo = (account_status: GetAccountStatus) => {
+    const risk_classification = account_status.risk_classification;
+
     const poa_status = account_status?.authentication?.document?.status || '';
     const poi_status = account_status?.authentication?.identity?.status || '';
 
     const idv_status = account_status?.authentication?.identity?.services?.idv?.status;
     const onfido_status = account_status?.authentication?.identity?.services?.onfido?.status;
     const manual_status = account_status?.authentication?.identity?.services?.manual?.status;
 
+    const is_authenticated_with_idv_photoid = account_status.status?.includes('authenticated_with_idv_photoid');
```

Comment:
```suggestion
    const is_authenticated_with_idv_photoid = account_status?.status?.includes('authenticated_with_idv_photoid');
```
Just a suggestion to make sure it don't get break as it was breaking test cases for me when I tried refactoring